tests/release/pipelines/rh_advisories.go

```diff
@@ -161,8 +161,13 @@ var _ = framework.ReleasePipelinesSuiteDescribe("e2e tests for rh-advisories pip
 				devFw = releasecommon.NewFramework(devWorkspace)
 				managedFw = releasecommon.NewFramework(managedWorkspace)
 
-				releaseCR, err = devFw.AsKubeDeveloper.ReleaseController.GetRelease("", snapshot.Name, devNamespace)
-				Expect(err).ShouldNot(HaveOccurred())
+				Eventually(func() error {
```

Comment:
Why did this change with the cosign change?

Reply:
The e2e-tests repo  migrated from openshift-ci to konflux-ci and it's to make the test pass there.